README.md

```diff
@@ -90,10 +90,10 @@ To run the FreeAPI project, follow these steps:
 5. Run the Docker Compose command:
 
 ```bash
-docker-compose up --build --attach backend
+docker-compose up --build --watch
```

Comment:
`unknown flag: --watch` getting this error

Reply:
re-iterating on it